src/DomainServices/Container/ContainerService.ts

```diff
@@ -445,34 +421,27 @@ export class ContainerService implements IContainerService {
   }
 
   public async getArchive(
-    userID: string,
     containerID: string,
     manuscriptID: string | null,
     token: string | null,
     options: ArchiveOptions
   ) {
-    const container = await this.containerRepository.getById(
-      containerID,
-      ContainerService.userIdForSync(userID)
-    )
+    const container = await this.containerRepository.getById(containerID)
 
     if (!container) {
       throw new MissingContainerError(containerID)
     }
 
-    if (this.isPublic(container)) {
-      return this.makeArchive(containerID, manuscriptID, options)
-    }
+    //This doesn't seem to be needed anymore
```

Comment:
@mnatsheh This is no longer needed, i commented it out until I confirm with you

Reply:
You can remove it.